weaver/samples/corda/corda-simple-application/clients/src/main/kotlin/com/cordaSimpleApplication/client/AssetTransferManager.kt

```diff
@@ -613,16 +614,19 @@ fun requestStateFromRemoteNetwork(
     val networkName = System.getenv("NETWORK_NAME") ?: "Corda_Network"
 
     try {
+        val relayOptions = RelayOptions(
+            useTlsForRelay = config["RELAY_TLS"]!!.toBoolean(),
+            relayTlsTrustStorePath = config["RELAY_TLSCA_TRUST_STORE"]!!,
+            relayTlsTrustStorePassword = config["RELAY_TLSCA_TRUST_STORE_PASSWORD"]!!,
+            tlsCACertPathsForRelay = config["RELAY_TLSCA_CERT_PATHS"]!!
+        )
         InteroperableHelper.interopFlow(
             proxy,
+            arrayOf(externalStateAddress),
             localRelayAddress,
-            externalStateAddress,
             networkName,
-            externalStateParticipants,
-            config["RELAY_TLS"]!!.toBoolean(),
-            config["RELAY_TLSCA_TRUST_STORE"]!!,
-            config["RELAY_TLSCA_TRUST_STORE_PASSWORD"]!!,
-            config["RELAY_TLSCA_CERT_PATHS"]!!
+            externalStateParticipants = externalStateParticipants,
+            relayOptions = relayOptions
```

Comment:
This function call seems to be fewer parameters than the function definition requires. Am I missing something?

Reply:
Yes this is working fine because of default arguements in SDK function, and passing named parameters (i.e. not depending on the order).